README.rst

```diff
@@ -210,6 +210,27 @@ Note that you can also create an automatically archived session, by passing in
 For more information on archiving, see the
 `OpenTok archiving <https://tokbox.com/opentok/tutorials/archiving/>`_ programming guide.
 
+Sending Signals
+~~~~~~~~~~~~~~~~~~~~~
+
+Once a Session is created, you can send signals to everyone in the session or to a specific connection. You can send a signal by calling the ``signal(session_id, data)`` method of the ``OpenTok`` class. The ``data`` parameter is a dictionary used to set the ``type``, ``data`` fields. Ỳou can also call the method with the parameter ``connection_id`` to send a signal to a specific connection ``signal(session_id, data, connection_id)``.
```

Comment:
@normanargueta since we are now calling it `payload`, can you also update the readme with that info?